PVEntityGenerator/app.config

```diff
@@ -17,4 +17,4 @@
       <remove invariant="MySql.Data.MySqlClient"/>
       <add name="MySQL Data Provider" invariant="MySql.Data.MySqlClient" description=".Net Framework Data Provider for MySQL" type="MySql.Data.MySqlClient.MySqlClientFactory, MySql.Data, Version=6.9.8.0, Culture=neutral, PublicKeyToken=c5687fc88969c44d"/>
     </DbProviderFactories>
-  </system.data><startup><supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1"/></startup></configuration>
+  </system.data><startup><supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.8"/></startup></configuration>
```

Comment:
it should not be required to update the dependency here in this PR - if required this should go to a separate PR.